packages/react-components/react-drawer/src/components/DrawerHeaderTitle/useDrawerHeaderTitle.ts

```diff
@@ -1,7 +1,24 @@
 import * as React from 'react';
-import { getNativeElementProps, slot } from '@fluentui/react-utilities';
+import { getNativeElementProps, slot, resolveShorthand } from '@fluentui/react-utilities';
 import type { DrawerHeaderTitleProps, DrawerHeaderTitleState } from './DrawerHeaderTitle.types';
-import { useDialogTitle_unstable } from '@fluentui/react-dialog';
+
+const getHeadingSlot = (props: DrawerHeaderTitleProps) => {
+  let heading = resolveShorthand(props.heading);
```

Comment:
```suggestion
  let heading = slot.resolveShorthand(props.heading);
```

If everything goes well `resolveShorthand` will be deprecated one day, just use `slotresolveShorthand` instead